conf.d/enhancd.fish

```diff
@@ -37,6 +37,8 @@ function __enhancd_install --on-event enhancd_install
     if not test -f "$ENHANCD_DIR/enhancd.log"
         touch "$ENHANCD_DIR/enhancd.log"
     end
+
+    alias $ENHANCD_COMMAND 'enhancd'
```

Comment:
@kidonng Why do we want to create an alias here, when we could use a function instead?

Reply:
I thought about that, but not sure if it worth the effort.